base_classes/NXprocess.nxdl.xml

```diff
@@ -43,6 +43,21 @@
     <field name="date" type="NX_DATE_TIME">
         <doc>Date and time of processing.</doc>
     </field>
+    <group type="NXregistration">
+        <doc>
+             Describes the operations of image registration
+        </doc>
+    </group>
+    <group type="NXdistortion">
+        <doc>
+             Describes the operations of image distortion correction
+        </doc>
+    </group>
+    <group type="NXcalibration">
+        <doc>
+             Describes the operations of calibration procedures, e.g. axis calibrations.
+        </doc>
+    </group>
```

Comment:
Remove these as they are subclasses of NXprocess

Reply:
done

Reply:
For the new added `group` elements, if the `doc` is a repeat of what is in the referenced base class, then the `doc` is not necessary here.

Reply:
These base classes are now extending `NXprocess` and thus are not in added to `NXprocess` anymore.